src/material/datepicker/date-range-input-parts.ts

```diff
@@ -238,6 +238,10 @@ export class MatStartDate<D> extends _MatDateRangeInputBase<D> implements CanUpd
     }
   }
 
+  protected _canEmitChangeEvent = (event: DateSelectionModelChange<DateRange<D>>): boolean => {
+    return event.source !== this && event.source !== this._rangeInput._endInput;
```

Comment:
Is the `event.source !== this` part necessary? The single date input doesn't check this

Reply:
It's not necessary since the check is inside the base class. It's a leftover from when I was trying something different. Removed.